qcodes_contrib_drivers/drivers/Lakeshore/Model_625.py

```diff
@@ -0,0 +1,523 @@
+import logging
+import time
+from typing import Union, Tuple
+
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import  Numbers, Enum
+
+log = logging.getLogger(__name__)
+
+class Lakeshore625(VisaInstrument):
+    """
+    Driver for the Lakeshore Model 625 superconducting magnet power supply.
+
+    This class uses T/A and A/s as units.
+
+    Args:
+        name (str): a name for the instrument
+        address (str): VISA address of the device
+        current_ramp_limit: A current ramp limit, in units of A/s
```

Comment:
This seems be called `field_ramp_rate` below

Reply:
True, my bad. One should specify the ramp rate when initializing the instrument.,